compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -1437,6 +1444,11 @@ object Types {
       case _ => true
     }
 
+    /** Is this (an alias of) the `scala.Null` type? */
+    final def isNull(given Context) =
+      isRef(defn.NullClass)
+      || classSymbol.name == tpnme.Null // !!! temporary kludge for being able to test without the explicit nulls PR
```

Comment:
Do we still need this second condition? The comment seems to indicate we no longer do.

Reply:
Removed